lib/BackgroundJobs/StorageCrawlJob.php

```diff
@@ -78,6 +93,30 @@ protected function run($argument): void {
 			$this->logger->error('Could not find storage root');
 			return;
 		}
+		
+		$qb = new CacheQueryBuilder($this->db, $this->systemConfig, $this->logger);
+		try {
+			$ignoreFiles = $qb->selectFileCache()
+				->andWhere($qb->expr()->eq('mimetype', $qb->createNamedParameter(15)))
+				->andWhere($qb->expr()->eq('name', $qb->createNamedParameter('.nomedia')))
+				->orWhere($qb->expr()->eq('name', $qb->createNamedParameter('.noimage')))
+				->executeQuery()->fetchAll();
+
+			$ignoreDir = [];
+			foreach ($ignoreFiles as $ignoreFile) {
+				$qb = new CacheQueryBuilder($this->db, $this->systemConfig, $this->logger);
+				$parent = $qb->selectFileCache()
+					->andWhere($qb->expr()->eq('mimetype', $qb->createNamedParameter(2)))
```

Comment:
the mimetype ids are different for every installation unfortunately. Can you try working around that similar to here: https://github.com/nextcloud/recognize/blob/master/lib/BackgroundJobs/StorageCrawlJob.php#L82